recipes-ni/proprietary/rcu-service/rcu-service-python-test-client_1.0.bb

```diff
@@ -6,7 +6,8 @@ require rcu-service-src.inc
 do_compile() {
 	python3 -m grpc_tools.protoc -I${S} --python_out=${S}/tests/python --grpc_python_out=${S}/tests/python ${S}/rcu-service.proto
 	python3 -m grpc_tools.protoc -I${S} --python_out=${S}/tests/pytest/rcu_pytest --grpc_python_out=${S}/tests/python ${S}/rcu-service.proto
-	python3 ${S}/tests/pytest/setup.py bdist_wheel -d ${S}/tests/pytest/dist
+	cd ${S}/tests/pytest
+	python3 setup.py bdist_wheel
```

Comment:
do we need `-d dist` ?

Reply:
I ran the command on my dev machine, and it automatically creates a dist folder in the same path